### PR TITLE
Update intro.py

### PR DESCRIPTION
**Pull request by dkangala**
In the "How to Use this Guide" section, if a person is "not" familiar, then the first sentence provides reference to a page for reading about other deep learning frameworks but there is a typo  in this first sentence. I am submitting this request to correct. Please find the sentence before and after correction for reference below.

Before => If you’re familiar with other deep learning frameworks, check out the 0. Quickstart first to quickly familiarize yourself with PyTorch’s API.

After => If you’re not familiar with other deep learning frameworks, check out the 0. Quickstart first to quickly familiarize yourself with PyTorch’s API.

Fixes #ISSUE_NUMBER

## Checklist

- [ ] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [ ] Only one issue is addressed in this pull request
- [ ] Labels from the issue that this PR is fixing are added to this pull request
- [ ] No unnecessary issues are included into this pull request.

### CODE REVIEW

**Review by malfet — CHANGES_REQUESTED** [review]
Chapter 0(Quickstart)  seems more advances than Chapter 1 (Tensors), so current wording is correct albeit confusing.

@subramen do you mind suggesting how to best reword it? Perhaps something like "If you are familiar with other deep learning feel free to browse an "Overview" chapter and skip the rest of the tutorial?

**Review by subramen — CHANGES_REQUESTED** [review]
This guide is meant to be a quick overview of the PyTorch API for people who are generally familiar with using deep learning frameworks, since it does not dive into the concepts.

I suggest rewording it to something like 
"If you're familiar with deep learning concepts, or perhaps you have used other deep learning frameworks before, feel free to browse the 0. Quickstart <quickstart_tutorial.html>`_ and skip the rest of the tutorial.

On the other hand, if you're new to deep learning, head right in... "

Thanks for improving the clarity on this guide
